easybuild/easyconfigs/e/einops/einops-0.3.2-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PythonPackage'
+
+name = 'einops'
+version = '0.3.2'
```

Comment:
There are newer versions available, could you use that for MONAI?

Also, please convert to PythonBundle since we're moving in that direction for all PythonPackages